### PR TITLE
Adding Features: Header/Footer/Map

### LINKED ISSUE #3: Develop Header with Navigation
#### Description
The task involves creating the website header based on the design provided in Figma. Additionally, implement navigation links that smoothly scroll the page to different sections upon clicking.

#### Tasks
- [x] Review the Figma design for the header component.
- [x] Create HTML/CSS structure for the header.
- [x] Implement navigation links to relevant sections of the site.
- [x] Test navigation functionality and smooth scrolling.
- [x] Ensure responsiveness and compatibility across devices.

Screen Shot 2023-12-05 at 9 46 56 PM

### LINKED ISSUE #4: Develop Footer with Linked 'Designed & Developed by Bizznest'
#### Description
Create a footer section for the website containing the text 'Designed & Developed by Bizznest' linked to Bizznest's website.

#### Tasks
- [x] Create HTML/CSS structure for the footer section.
- [x] Ensure the 'Designed & Developed by Bizznest' text is appropriately linked to [bizznest.digitalnest.org](https://bizznest.digitalnest.org).
- [x] Ensure the footer is responsive and compatible with various screen sizes.

### PR DESCRIPTION
**Pull request by elimanzo**
- Adding footer
- Adding header

Submitting a Draft PR for now of the progress. I still need to add the map

image

Currently:
Closes #3
Closes #4

### CODE REVIEW

**Comment by alexrdrgz** [comment]
It's not good practice to merge 3 issues with one pull request. Let's split these into their own.

Right now it seems redundant because it's a small set of features but I just want to start getting us used to best practices.

**Comment by alexrdrgz** [comment]
Also the commit messages aren't following the standards laid out in the readme.md.

**Review by alexrdrgz — CHANGES_REQUESTED** [review]
Comments made below.

Also naming conventions for branch names should be "initals-issue-nuimber-issue description"

File: index.html [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    <script src="https://unpkg.com/leaflet/dist/leaflet.js"></script>
+    <script src="script.js"></script>
+  </head>
+  <body>
+    <header>
+      <div>
+        <h3>John Doe</h3>
+      </div>
+      <nav>
+        <a href="#projects">Projects</a>
```
Comment (alexrdrgz) [inline index.html:18]: Lets use semantic html for navigation <ul> <li> tags. This makes the navigation more accessible as well.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+        <a href="#contacts">Contacts</a>
+      </nav>
+    </header>
+    <div id="map"></div>
+    <footer>
+      <p>
+        <a href="https://bizznest.digitalnest.org/" target="_blank"
+          >Designed & Built by Bizznest</a
+        >
```
Comment (alexrdrgz) [inline index.html:27]: Let's fix this line break

File: index.html [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+    <header>
+      <div>
+        <h3>John Doe</h3>
+      </div>
+      <nav>
+        <a href="#projects">Projects</a>
+        <a href="#contacts">Contacts</a>
+      </nav>
+    </header>
+    <div id="map"></div>
```
Comment (alexrdrgz) [inline index.html:22]: This map should be in the <main></main> section of the html. Let's refactor this when you create a new PR

File: script.js [hunk 4]
```diff
@@ -0,0 +1,7 @@
+document.addEventListener('DOMContentLoaded', function () {
```
Comment (alexrdrgz) [inline script.js:1]: For code like this that's not super readable, let's add a short comment about what this function does

**Review by alexrdrgz — CHANGES_REQUESTED** [review]
Some more comments for when you break this up into different PR's

**Comment by elimanzo** [comment]
> Some more comments for when you break this up into different PR's

@alexrdrgz thanks for the feedback I'll apply these changes and create new PRs snd Branches on Tuesday 😊

File: index.html (hunk 1 above)
Comment (elimanzo) [reply index.html:18]: I assumed <nav> is a semantic html tag? Did you mean that John Doe should've also been in the nav?

File: index.html (hunk 3 above)
Comment (alexrdrgz) [reply index.html:22]: I fixed my comment above, the `<main></main>` wrapper

File: index.html (hunk 1 above)
Comment (alexrdrgz) [reply index.html:18]: Fixed this comment too! I meant `<ul><li></li></ul>`

**Comment by elimanzo** [comment]
Closing PR: Broken up into 3 PRs(#15 #16 #17)

**Pull request closed without merging**